products/metadata/metadata_template.html

```diff
@@ -0,0 +1,239 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Metadata</title>
+</head>
+
+<body>
+    <h1>{{ metadata.display_name }}</h1>
+
+    <h2><strong>Description:</strong> </h2>
+    <p>{{ metadata.description }}</p>
+
+    <h2><strong>Tags:</strong></h2>
+    <ul>
+        {% for tag in metadata.tags %}
+        <li>{{ tag }}</li>
+        {% endfor %}
+    </ul>
+
+    <h2><strong>Each row is a: </strong> </h2>
+    <p>{{ metadata.each_row_is_a }}</p>
+
+    <h2><strong>Destinations: </strong> </h2>
```

Comment:
Let's remove destination info for now. It might be interested to include something like that in the future, but I'll have to think about it.